util/chunk/chunk_util_test.go

```diff
@@ -51,6 +51,8 @@ func getChk(isLast3ColTheSame bool) (*Chunk, *Chunk, []bool) {
 }
 
 func TestCopySelectedJoinRows(t *testing.T) {
+	t.Parallel()
+
```

Comment:
you may make use of testify checker to simplify bare go testing `if` block.

Reply:
done